hugegraph-api/src/main/java/com/baidu/hugegraph/api/API.java

```diff
@@ -45,7 +45,7 @@
 
 public class API {
 
-    protected static final Logger LOG = Log.logger(RestServer.class);
+    protected static final Logger LOG = Log.logger(API.class);
```

Comment:
Also, it would be nice if the log-framework could support a unified approach like this (I don't know if there is a way) :
`Logger LOG = Log.logger(this);